package.json

```diff
@@ -13,7 +13,7 @@
     "start": "npm test -- --watch",
     "clean": "rimraf dist",
     "build": "tsc",
-    "test": "jest",
+    "test": "npm run build && jest",
```

Comment:
Running build before jest is not required with ts-jest.

Reply:
So, relative to how I am building stuff, this is; basically I wanted to be able to ask the compiled / for-publication version to build things at the command-line, and `dist/src` doesn't exist at that point.

More comments below.